### DISCUSSION TITLE
How to enable aggrid tree view

### DISCUSSION THREAD

**Discussion by Janet-Tester** [zauberzeug/nicegui#1525] [Q&A] [answered]
### Question

I failed to show the data in the aggrid as tree view, and I also not sure my way is correct. 

Below is my code: 

```py
from nicegui import ui

grid_header = [{'headerName': 'Name', 'field':'Name', 'checkboxSelection': True, 'headerCheckboxSelection': True, 'showDisabledCheckboxes': True },
          {'headerName': 'Age', 'field':'Age'},
          {'headerName': 'Color', 'field':'Color'},
          ]

row_data = [
    {'Name': ["Jack"], 'Age': '18', 'Color':"Blue"}, 
    {'Name': ["Jack", "Paul"], 'Age': '15', 'Color':"Red"}, 
    {'Name': ["Jack", "Paul", "Amy"], 'Age': '20', 'Color':"Black"}]

grid = ui.aggrid(
    options = {
    'columnDefs':[
        {'field': "Age"},
        {'field': "Color"},
    ],
    'autoGroupColumnDef': {
    'headerName': 'Name',
    'minWidth': 300,
    'cellRendererParams': {
    'suppressCount': True,
},
},
    'defaultColDef': {'flex': 1},
    'columnDefs': grid_header,
    'rowData': row_data,
    'rowSelection': 'multiple',
    'treeData': True,
    'getDataPath': True
})

ui.run(port=8080)
```

**Comment by falkoschindler** [comment, marked as answer]
According to the JavaScript console, row grouping seems to be an AG Grid enterprise feature:
> AG Grid: unable to use Tree Data as 'ag-grid-enterprise' has not been loaded. Check you are using the Enterprise bundle:
>
>     <script src="https://cdn.jsdelivr.net/npm/ag-grid-enterprise@AG_GRID_VERSION/dist/ag-grid-enterprise.min.js"></script>
>
> For more info see: https://ag-grid.com/javascript-data-grid/getting-started/#getting-started-with-ag-grid-enterprise

**Reply by Janet-Tester** [reply]
Thanks for your answer. That's really helpful.
